src/components/select/filterable-select/filterable-select-test.stories.tsx

```diff
@@ -695,3 +695,38 @@ export const SelectionConfirmed = () => {
     </>
   );
 };
+
+export const FilterableSelectWithDisabledOption = () => {
```

Comment:
@harpalsingh @ljemmo currently the input field will autofill with the first Option available when typing, even if it is a disabled Option. Should we prevent this from happening if the first match is not available or completely excluding it from the filtered list as Ed suggested above?

Reply:
It should not select or auto fill a disabled option, because a user would not be able to do it with keyboard/mouse either. As above, will discuss if we need to show in filter list when type ahead filter is being applied.